db/schema.sql

```diff
@@ -16,4 +16,5 @@ DEFINE FIELD url ON TABLE link TYPE string;
 DEFINE FIELD title ON TABLE link TYPE string;
 DEFINE FIELD note ON TABLE link TYPE string;
 DEFINE FIELD user ON TABLE link TYPE record (user);
+DEFINE FIELD bookmarked_at ON TABLE link TYPE datetime DEFAULT time::now();
```

Comment:
I actually ran into an issue implementing this initially where it passed the tests but failed when actually using it because this schema only applies to tests, so the default I'm setting here wasn't getting used so I had to explicitly pass it in `create_link`.

Reply:
I think that might be because currently the dev setup requires manually adding the schema definition (SurrealDB supports schemaless as well). I don't yet know if there is a good easy to use library for migrations yet so I currently just manually run these define statements. Either way I think I prefer the explicit assignment while creating the link!

Reply:
I started playing around with this a bit as I was looking into this problem which seemed pretty easy to integrate: https://github.com/Odonno/surrealdb-migrations/

Reply:
Yeah that would be greatly appreciated if you have time! :pray: